packages/alfa-dom/src/get-tab-sequence.ts

```diff
@@ -0,0 +1,46 @@
+import { getTabIndex } from "./get-tab-index";
+import { traverseNode } from "./traverse-node";
+import { Element, NodeType } from "./types";
+
+function binaryIndexSearch(array: Array<Element>, index: number) {
```

Comment:
I'd call this function `indexWithin()` (in line with `indexOf()`) and make the second parameter of type `Element`. That would make it read as "Given this array of elements, get the index that this element would have within that array".